tslint.json

```diff
@@ -0,0 +1,18 @@
+{
+  "extends": "tslint-config-airbnb",
+  "rules": {
+    "no-increment-decrement": false,
+    "import-name": false,
+    "max-line-length": false,
+    "function-name": false,
+    "variable-name": [
+      true,
+      "ban-keywords",
+      "check-format",
+      "allow-snake-case",
+      "allow-leading-underscore",
+      "allow-pascal-case"
+    ],
+    "trailing-comma": false
+  }
+}
```

Comment:
Please add a line in the end of the file

Reply:
FINE